choc-pizza.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Chocolate Pizza
+		</title>
+		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
+		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand' rel='stylesheet' type='text/css' />
+	</head>
+	<body class="container_16 clearfix">
+		<header id="top-header" class="grid_15">
+			<div id="top-logo" class="toplogo">
+				<img src="img/logo.png" />
```

Comment:
Dexter,

There is a blog called HTML5Doctor that spills many thousands of words on
semantic content. There is a post that seems to think the "social links"
bar should be classified as an aside. I was hoping you would address this
part of it. I'm not trying to argue or disagree with you, I'm just trying
to gain a fuller understanding of this topic. I think this is something
that will be very important for me to understand if I intend to make web UI
my area of focus.

I would note that if you make the social links a section, the outliner tool
will output an untitled section, which doesn't seem to add much semantic
value. From what I've read, section should almost always only be used when
there is a natural heading, to avoid outputting untitled sections in the
outliner. Untitled sections could possibly be div's. Please advise. Thank
you.

http://html5doctor.com/your-questions-16/

"A list of share icons isn’t <nav> for two separate reasons.

First, they aren’t meant to take you somewhere else. They’re designed to
perform an action of tweeting/”liking” a link to a page.

Second, and most importantly, they’re not navigation around your content.
As the spec says
<http://dev.w3.org/html5/spec/sections.html#the-nav-element> “Not all
groups of links on a page need to be in a nav element — only sections that
consist of major navigation blocks are appropriate for the nav element.”

As a list of share icons is tangential to your content, I’d probably use the
<aside> element <http://dev.w3.org/html5/spec/sections.html#the-nav-element>:
“The aside element represents a section of a page that consists of content
that is tangentially related to the content around the aside element, and
which could be considered separate from that content. Such sections are
often represented as sidebars in printed typography”. For more details,
check out our <aside> revisited <http://html5doctor.com/aside-revisited/>
article.

Social links are the kind of thing that could be cut from a feed reader
without diminishing the value of the content. As an additional note,
remember that you can also place an <aside> at the end of an article, like
a footer. In spite of its name, <aside> doesn’t have to be a sidebar"

On Mon, Jan 19, 2015 at 2:38 PM, Jay Springate <jay.springate@gmail.com>
wrote:

> I had a reason at the time for aligning everything along grid_15, but now
> I can't remember what it was. I will change those to grid_16 and see what
> happens and make adjustments as necessary. Thank you.
>
> On Mon, Jan 19, 2015 at 2:33 PM, Dexter Lesaca <notifications@github.com>
> wrote:
>
>> In choc-pizza.html
>> <https://github.com/jayspringate/chocolate-pizza/pull/2#discussion-diff-23191516>
>> :
>>
>> > @@ -0,0 +1,119 @@
>> > +<!DOCTYPE html>
>> > +<html>
>> > +	<head>
>> > +		<title>
>> > +			Chocolate Pizza
>> > +		</title>
>> > +		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
>> > +		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
>> > +		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
>> > +		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>' rel='stylesheet' type='text/css' />
>> > +	</head>
>> > +	<body class="container_16 clearfix">
>> > +		<header id="top-header" class="grid_15">
>> > +			<div id="top-logo" class="toplogo">
>> > +				<img src="img/logo.png" />
>>
>> Both are acceptable. If you’re going to apply the image, use the
>> background image property in css. Just be sure to apply it to an existing
>> element, not create a new one. — Sent from Mailbox
>> … <#14b0458efe9a57c0_14b0454984a19435_>
>> On Mon, Jan 19, 2015 at 2:32 PM, Jay Springate <notifications@github.com>
>> wrote: >> @@ -0,0 +1,119 @@ >> +<!DOCTYPE html> >> +<html> >> + <head> >> +
>> <title> >> + Chocolate Pizza >> + </title> >> + <link
>> href="css/reset-choc.css" type="text/css" rel="stylesheet" /> >> + <link
>> href="css/style-choc.css" type="text/css" rel="stylesheet" /> >> + <link
>> href="css/960_16_col.css" type="text/css" rel="stylesheet" /> >> + <link
>> href='
>> http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand
>> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>'
>> rel='stylesheet' type='text/css' /> >> + </head> >> + <body
>> class="container_16 clearfix"> >> + <header id="top-header"
>> class="grid_15"> >> + <div id="top-logo" class="toplogo"> >> + <img
>> src="img/logo.png" /> > Also, I never figured out the ideal way to
>> implement that slash border. I > ended up screenshotting it and including
>> it as an image in the html. I > realize this is incorrect because it
>> includes styling in the html. Would > you prefer I approximate that border
>> using one of the existing CSS styles, > or that I include the image purely
>> through CSS somehow? Or is there another > solution? > On Mon, Jan 19, 2015
>> at 2:25 PM, Dexter Lesaca <notifications@github.com> > wrote: > In
>> choc-pizza.html > <
>> https://github.com/jayspringate/chocolate-pizza/pull/2#discussion-diff-23191073>
>> > : > > > @@ -0,0 +1,119 @@ > > +<!DOCTYPE html> > > +<html> > > + <head> >
>> > + <title> > > + Chocolate Pizza > > + </title> > > + <link
>> href="css/reset-choc.css" type="text/css" rel="stylesheet" /> > > + <link
>> href="css/style-choc.css" type="text/css" rel="stylesheet" /> > > + <link
>> href="css/960_16_col.css" type="text/css" rel="stylesheet" /> > > + <link
>> href='
>> http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand
>> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>
>> <
>> http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>
>> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand%3E>'
>> rel='stylesheet' type='text/css' /> > > + </head> > > + <body
>> class="container_16 clearfix"> > > + <header id="top-header"
>> class="grid_15"> > > + <div id="top-logo" class="toplogo"> > > + <img
>> src="img/logo.png" /> > > "/" is not necessary for self-closing tags. > > —
>> > Reply to this email directly or view it on GitHub > <
>> https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191073>.
>> > --- Reply to this email directly or view it on GitHub:
>> https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191424
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191516>.
>>
>
>